src/Elcodi/Template/StoreTemplateBundle/Resources/views/Pages/checkout-payment.html.twig

```diff
@@ -69,7 +77,9 @@
 
     {{ parent() }}
 
+    {% if enabledPaymentMethods['paymill'] %}
     {{ paymill_scripts() }}
```

Comment:
We are indenting in the same level that the html if I'm not wrong, right @tonipinel ?